src/utils/keyRotation.ts

```diff
@@ -12,18 +12,24 @@ export function shouldRotateKey(date: Date, env: Bindings): boolean {
 }
 
 function matchCronTime(cronString: string, date: Date): boolean {
+	// Set seconds and milliseconds to 0 to round to the nearest minute
+	date.setUTCSeconds(0, 0);
```

Comment:
TIL that you can pass two parameters to [`setUTCSeconds`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/setUTCSeconds#parameters)

maybe point to the documentation. I was expecting two calls somehow, one for seconds, one for milliseconds